api.go

```diff
@@ -474,45 +474,45 @@ func DeleteContainer(podID, containerID string) (*Container, error) {
 
 // StartContainer is the virtcontainers container starting entry point.
 // StartContainer starts an already created container.
-func StartContainer(podID, containerID string) (*Container, error) {
+func StartContainer(podID, containerID string) (*Pod, *Container, error) {
```

Comment:
We already did the same (returning the Pod) for the EnterContainer() so that our runtime 3.0 was able to get the pod URL. Either with keep this solution or we need to change EnterContainer also.